rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -191,6 +191,48 @@ def _add_table_id(self, record_batch: RecordBatch, table_id: str) -> RecordBatch
         # Create new record batch with updated schema
         return RecordBatch.from_arrays(record_batch.columns, schema=schema)
 
+    def set_application_blueprint(
```

Comment:
We should have an example/test notebook actually use this API

Reply:
added an example notebook